docs/usage.rst

```diff
@@ -19,6 +19,16 @@ By default the import process will start by running a simple graphical editor,
 allowing you to input metadata required or recommended by the NWB format,
 but that is not available within the Labview data folder.
 
+``metadata2nwb`` creates a 'bare bones' NWB file by loading metadata describing
```

Comment:
```suggestion
``metadata2nwb`` creates a minimal NWB file by loading metadata describing
```
I would rather the "bare-bones" didn't stick but couldn't come up with a better alternative before.